# Russian translation for backintime
# Copyright (c) 2009 Rosetta Contributors and Canonical Ltd 2009
# This file is distributed under the same license as the backintime package.
# FIRST AUTHOR <EMAIL@ADDRESS>, 2009.
#
msgid ""
msgstr ""
"Project-Id-Version: backintime\n"
"Report-Msgid-Bugs-To: FULL NAME <EMAIL@ADDRESS>\n"
"POT-Creation-Date: 2013-11-09 22:34+0100\n"
"PO-Revision-Date: 2013-07-22 14:24+0000\n"
"Last-Translator: Anton Kolochencow <Unknown>\n"
"Language-Team: Russian <ru@li.org>\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Plural-Forms: nplurals=3; plural=n%10==1 && n%100!=11 ? 0 : n%10>=2 && "
"n%10<=4 && (n%100<10 || n%100>=20) ? 1 : 2;\n"
"X-Launchpad-Export-Date: 2013-12-10 17:48+0000\n"
"X-Generator: Launchpad (build 16869)\n"

#: ../../gnome/app.py:1126 ../../kde4/app.py:1013
msgid "View the current disk content"
msgstr "Просмотр содержимого текущего диска"

#: ../../gnome/settingsdialog.py:1173 ../../kde4/settingsdialog.py:1292
msgid "Include folder"
msgstr "Включить каталог"

#: ../../gnome/settingsdialog.py:1257 ../../kde4/settingsdialog.py:1261
msgid "Exclude folder"
msgstr "Исключить каталог"

#: ../../kde4/settingsdialog.py:408 ../../kde4/settingsdialog.py:450
#: ../../gnome/settingsdialog.glade:1427
msgid "Add folder"
msgstr "Добавить каталог"

#: ../../kde4/settingsdialog.py:460 ../../gnome/settingsdialog.glade:1905
msgid "Auto-remove"
msgstr "Автоматическое удаление"

#: ../../common/config.py:72
msgid "Disabled"
msgstr "Отключён"

#: ../../common/config.py:73
msgid "At every boot/reboot"
msgstr "При каждой загрузке системы"

#: ../../common/config.py:74
msgid "Every 5 minutes"
msgstr "Каждые 5 минут"

#: ../../common/config.py:75
msgid "Every 10 minutes"
msgstr "Каждые 10 минут"

#: ../../common/config.py:76
msgid "Every 30 minutes"
msgstr "Каждые 30 минут"

#: ../../common/config.py:77
msgid "Every hour"
msgstr "Каждый час"

#: ../../common/config.py:78
msgid "Every 2 hours"
msgstr "Каждые 2 часа"

#: ../../common/config.py:79
msgid "Every 4 hours"
msgstr "Каждые 4 часа"

#: ../../common/config.py:80
msgid "Every 6 hours"
msgstr "Каждые 6 часов"

#: ../../common/config.py:81
msgid "Every 12 hours"
msgstr "Каждые 12 часов"

#: ../../common/config.py:82
msgid "Custom Hours"
msgstr ""

#: ../../common/config.py:83
msgid "Every Day"
msgstr "Ежедневно"

#: ../../common/config.py:84
msgid "Daily (anacron)"
msgstr ""

#: ../../common/config.py:85
msgid "When drive get connected (udev)"
msgstr ""

#: ../../common/config.py:86
msgid "Every Week"
msgstr "Каждую неделю"

#: ../../common/config.py:87
msgid "Every Month"
msgstr "Каждый месяц"

#: ../../common/config.py:91
msgid "Day(s)"
msgstr "Дней (Дня)"

#: ../../common/config.py:92
msgid "Week(s)"
msgstr "недель"

#: ../../common/config.py:93
msgid "Year(s)"
msgstr "Лет (Года)"

#: ../../common/config.py:100
msgid " EXPERIMENTAL!"
msgstr " ЭКСПЕРИМЕНТАЛЬНО!"

#: ../../common/config.py:103
msgid "Local"
msgstr ""

#: ../../common/config.py:104
msgid "SSH"
msgstr "SSH"

#: ../../common/config.py:104 ../../common/config.py:106
#: ../../gnome/settingsdialog.py:475 ../../kde4/settingsdialog.py:1325
msgid "SSH private key"
msgstr "личный ключ SSH"

#: ../../common/config.py:105
msgid "Local encrypted"
msgstr ""

#: ../../common/config.py:105 ../../common/config.py:106
msgid "Encryption"
msgstr ""

#: ../../common/config.py:106
msgid "SSH encrypted"
msgstr "SSH зашифровано"

#: ../../common/config.py:112
msgid "Default"
msgstr "По умолчанию"

#: ../../common/config.py:113
msgid "AES128-CTR"
msgstr "AES128-CTR"

#: ../../common/config.py:114
msgid "AES192-CTR"
msgstr "AES192-CTR"

#: ../../common/config.py:115
msgid "AES256-CTR"
msgstr "AES256-CTR"

#: ../../common/config.py:116
msgid "ARCFOUR256"
msgstr "ARCFOUR256"

#: ../../common/config.py:117
msgid "ARCFOUR128"
msgstr "ARCFOUR128"

#: ../../common/config.py:118
msgid "AES128-CBC"
msgstr "AES128-CBC"

#: ../../common/config.py:119
msgid "3DES-CBC"
msgstr "3DES-CBC"

#: ../../common/config.py:120
msgid "Blowfish-CBC"
msgstr "Blowfish-CBC"

#: ../../common/config.py:121
msgid "Cast128-CBC"
msgstr "Cast128-CBC"

#: ../../common/config.py:122
msgid "AES192-CBC"
msgstr "AES192-CBC"

#: ../../common/config.py:123
msgid "AES256-CBC"
msgstr "AES256-CBC"

#: ../../common/config.py:124
msgid "ARCFOUR"
msgstr "ARCFOUR"

#: ../../common/config.py:129
msgid "Main profile"
msgstr "Основной профиль"

#: ../../common/config.py:256 ../../common/config.py:275
#: ../../common/config.py:286 ../../common/config.py:291
#, python-format
msgid "Profile: \"%s\""
msgstr "Профиль: \"%s\""

#: ../../common/config.py:256
msgid "Snapshots folder is not valid !"
msgstr "Каталог резервных копий задан неверно!"

#: ../../common/config.py:275
msgid "You must select at least one folder to backup !"
msgstr "Вы должны выбрать хотя бы одну папку для резервного копирования!"

#: ../../common/config.py:286
msgid "You can't include backup folder !"
msgstr "Вы не можете добавить папку для резервного копирования!"

#: ../../common/config.py:291
msgid "You can't include a backup sub-folder !"
msgstr "Вы не можете включить в архив вложенные каталоги"

#: ../../common/config.py:361
#, python-format
msgid "%s is not a folder !"
msgstr "%s не папка !"

#: ../../common/config.py:376 ../../common/config.py:389
#, python-format
msgid ""
"Can't write to: %s\n"
"Are you sure you have write access ?"
msgstr ""
"Не удаётся произвести запись в: %s\n"
"Вы уверены, что имеете право на запись?"

#: ../../common/config.py:1286
msgid ""
"Can't find crontab.\n"
"Are you sure cron is installed ?\n"
"If not you should disable all automatic backups."
msgstr ""
"Невозможно найти crontab\n"
"Вы уверены, что cron установлен?\n"
"Если не установлен, вы должны отключить автоматическое резервное копирование."

#: ../../common/config.py:1338
#, python-format
msgid "Schedule udev doesn't work with mode %s"
msgstr ""

#: ../../common/config.py:1342
#, python-format
msgid "Couldn't find UUID for \"%s\""
msgstr ""

#: ../../common/config.py:1379
msgid "Failed to remove udev rules"
msgstr ""

#: ../../common/config.py:1385
msgid "Failed to create udev rules"
msgstr ""

#: ../../common/config.py:1420
msgid ""
"Can't find anacron.\n"
"Are you sure anacron is installed ?\n"
"If not you should disable daily anacron backups."
msgstr ""

#: ../../common/config.py:1439
msgid "Please provide your sudo password to install the udev rule."
msgstr ""

#: ../../common/config.py:1445
msgid "Please provide your sudo password to remove unused udev rules."
msgstr ""

#: ../../common/configfile.py:253 ../../common/configfile.py:315
#, python-format
msgid "Profile \"%s\" already exists !"
msgstr "Профиль \"%s\" уже существует!"

#: ../../common/configfile.py:284
msgid "You can't remove the last profile !"
msgstr "Невозможно удалить последний профиль!"

#: ../../common/encfstools.py:95
#, python-format
msgid ""
"Can't mount '%(command)s':\n"
"\n"
"%(error)s"
msgstr ""
"Не могу смонтировать '%(command)s':\n"
"\n"
"%(error)s"

#: ../../common/encfstools.py:105
#, python-format
msgid "Can't unmount encfs %s"
msgstr "Не могу отмонтировать encfs %s"

#: ../../common/encfstools.py:149
msgid "Config for encrypted folder not found."
msgstr ""

#: ../../common/encfstools.py:153
msgid ""
"\n"
"Create a new encrypted folder?"
msgstr ""
"\n"
"Создать новую зашифрованную папку?"

#: ../../common/encfstools.py:154
msgid "Cancel"
msgstr "Отмена"

#: ../../common/encfstools.py:157
msgid "Please confirm password"
msgstr "Пожалуйста, подтвердите пароль"

#: ../../common/encfstools.py:161
msgid "Password doesn't match"
msgstr ""

#: ../../common/encfstools.py:166
msgid "encfs not found. Please install e.g. 'apt-get install encfs'"
msgstr ""
"encfs не найдена. Вы можете её установить, набрав в консоли 'apt-get install "
"encfs'"

#: ../../common/encfstools.py:174 ../../common/sshtools.py:207
#, python-format
msgid ""
"%(user)s is not member of group 'fuse'.\n"
" Run 'sudo adduser %(user)s fuse'. To apply changes logout and login again.\n"
"Look at 'man backintime' for further instructions."
msgstr ""
"%(user)s не является членом группы 'fuse'.\n"
" Запустите 'sudo adduser %(user)s fuse'. Чтобы применить изменения, выйдите "
"и войдите снова.\n"
"Выполните 'man backintime' для дальнейших инструкций."

#: ../../common/encfstools.py:189
msgid ""
"encfs version 1.7.2 and before has a bug with option --reverse. Please "
"update encfs"
msgstr ""
"encfs версии 1.7.2 и ранние имеют баг с опцией --reverse. Пожалуйста, "
"обновите encfs."

#: ../../common/encfstools.py:453 ../../common/encfstools.py:480
#: ../../common/snapshots.py:982 ../../common/snapshots.py:1288
#: ../../kde4/app.py:82 ../../gnome/mainwindow.glade:43
msgid "Take snapshot"
msgstr "Сделать резервную копию"

#: ../../common/mount.py:195
#, python-format
msgid ""
"Hash collision occurred in hash_id %s. Incrementing global value "
"hash_collision and try again."
msgstr ""

#: ../../common/mount.py:246
#, python-format
msgid "mountpoint %s not empty."
msgstr ""

#: ../../common/mount.py:286
msgid "Mountprocess lock timeout"
msgstr ""

#: ../../common/password.py:425
#, python-format
msgid "Profile '%(profile)s': Enter password for %(mode)s: "
msgstr ""

#: ../../common/snapshots.py:171 ../../gnome/app.py:1127
#: ../../kde4/app.py:1014
msgid "Now"
msgstr "Сейчас"

#: ../../common/snapshots.py:182 ../../gnome/gnometools.py:37
msgid "WITH ERRORS !"
msgstr "С ОШИБКАМИ!"

#: ../../common/snapshots.py:282
msgid ""
"### This log has been decoded with automatic search pattern\n"
"### If some paths are not decoded you can manually decode them with:\n"
msgstr ""

#: ../../common/snapshots.py:465
msgid "FAILED"
msgstr "НЕУСПЕШНО"

#: ../../common/snapshots.py:564
msgid "Restore permissions:"
msgstr ""

#: ../../common/snapshots.py:739
msgid ""
"Back In Time changed its backup format.\n"
"\n"
"Your old snapshots can be moved according to this new format. OK?"
msgstr ""
"В Back In Time сменился используемый формат.\n"
"\n"
"Ваши старые снимки могут быть преобразованы в соответствии с новым форматом. "
"Преобразовать?"

#: ../../common/snapshots.py:749
#, python-format
msgid ""
"%s profiles found. \n"
"\n"
"The new backup format supports storage of different users and profiles on "
"the same location. Do you want the same location for both profiles? \n"
"\n"
"(The program will still be able to discriminate between them)"
msgstr ""
"%s профилей найдено. \n"
"\n"
"Новый формат сохранений резервных копий поддерживает хранилище различных "
"пользователей в одном месте. Хотите ли Вы использовать одинаковое размещение "
"для обоих пользователей?\n"
"(Программа сможет различать профили)"

#: ../../common/snapshots.py:752
msgid ""
"No profiles are found. Will have to update to profiles first, please restart "
"Back In Time"
msgstr ""
"Профили не найдены. Необходимо сначала обновить профили, пожалуйста, "
"перезапустите Back In Time"

#: ../../common/snapshots.py:803
#, python-format
msgid ""
"%(snapshots_left)s\n"
"of profile %(profile_id)s are not moved to their new location\n"
"Do you want to proceed?\n"
"(Back In Time will be able to continue taking snapshots, however the "
"remaining snapshots will not be considered for automatic removal)\n"
"\n"
"If not Back In Time will restore former settings for this profile, however "
"cannot continue taking snapshots"
msgstr ""

#: ../../common/snapshots.py:812
#, python-format
msgid ""
"Former settings of profile %s are restored.\n"
"Back In Time cannot continue taking new snapshots.\n"
"\n"
"You can manually move the snapshots, \n"
"if you are done restart Back In Time to proceed"
msgstr ""
"Предыдущие настройки профиля %s восстановлены.\n"
"Back In Time не может продолжать создавать новые снимки.\n"
"\n"
"Вы можете вручную переместить снимки, \n"
"если вы сделали, что хотели, то перезапустите Back In Time, чтобы продолжить."

#: ../../common/snapshots.py:825
msgid ""
"Update was successful!\n"
"\n"
"Back In Time will continue taking snapshots again as scheduled"
msgstr ""
"Обновление завершено успешно!\n"
"\n"
"Back In Time продолжит выполнять резервное копирование по расписанию"

#: ../../common/snapshots.py:830
msgid ""
"Are you sure you do not want to move your old snapshots?\n"
"\n"
"\n"
"If you do, you will not see these questions again next time, Back In Time "
"will continue making snapshots again, but smart-remove cannot take your old "
"snapshots into account any longer!\n"
"\n"
"If you do not, you will be asked again next time you start Back In Time."
msgstr ""
"Вы уверены что не хотите переместить старые снимки?\n"
"\n"
"\n"
"Если уверены, вы не увидите эти вопросы при следующем запуске, Back In Time "
"продоллит созданиеснимков, но smart-remove больше не может  использовать "
"старые снимки в учетной записи!\n"
"\n"
"Если не уверены, Back In Time повторно спросит Вас при следующем запуске."

#: ../../common/snapshots.py:839
msgid "Back In Time will continue taking snapshots again as scheduled"
msgstr "Back In Time продолжит выполнять резервное копирование по расписанию"

#: ../../common/snapshots.py:841
msgid ""
"Back In Time still cannot continue taking new snapshots.\n"
"Restart Back In Time to see the questions again"
msgstr ""
"Back In Time все еще не может продолжить создание новых снимков.\n"
"Перезапустите Back In Time для повторения вопросов"

#: ../../common/snapshots.py:913
msgid ""
"Can't find snapshots folder.\n"
"If it is on a removable drive please plug it."
msgstr ""
"Невозможно найти каталог резервных копий!\n"
"Если он находится на съёмном диске, пожалуйста, подключите его."

#: ../../common/snapshots.py:915
#, python-format
msgid "Waiting %s second."
msgid_plural "Waiting %s seconds."
msgstr[0] "Ждать %s сек."
msgstr[1] "Ждать %s сек."
msgstr[2] "Ждать %s сек."

#: ../../common/snapshots.py:941
#, python-format
msgid "Failed to take snapshot %s !!!"
msgstr "Не удалось создать резервную копию %s !!!"

#: ../../common/snapshots.py:950
msgid "Finalizing"
msgstr "Окончание"

#: ../../common/snapshots.py:1088
#, python-format
msgid "Can't create folder: %s"
msgstr "Не могу создать каталог: %s"

#: ../../common/snapshots.py:1107
msgid "..."
msgstr "..."

#: ../../common/snapshots.py:1134
#, python-format
msgid "Can't remove folder: %s"
msgstr "Не могу удалить каталог: %s"

#: ../../common/snapshots.py:1219
#, python-format
msgid "Compare with snapshot %s"
msgstr "Сравнить с резервной копией %s"

#: ../../common/snapshots.py:1248
msgid "Create hard-links"
msgstr "Создать жёсткие ссылки"

#: ../../common/snapshots.py:1335
msgid "Save config file ..."
msgstr "Сохранить конфигурационный файл..."

#: ../../common/snapshots.py:1341
msgid "Save permission ..."
msgstr "Сохранить разрешение ..."

#: ../../common/snapshots.py:1442
#, python-format
msgid "Can't rename %(new_path)s to %(path)s"
msgstr ""

#: ../../common/snapshots.py:1562
msgid "Remove old snapshots"
msgstr "Удалить старые резервные копии"

#: ../../common/snapshots.py:1586 ../../kde4/settingsdialog.py:501
#: ../../gnome/settingsdialog.glade:1660
msgid "Smart remove"
msgstr "Умное удаление"

#: ../../common/snapshots.py:1591
msgid "Try to keep min free space"
msgstr "Стараться сохранять минимальное свободное место"

#: ../../common/snapshots.py:1621
#, python-format
msgid "Try to keep min %d%% free inodes"
msgstr ""

#: ../../common/sshtools.py:106
#, python-format
msgid "Can't mount %s"
msgstr "Не могу смонтировать %s"

#: ../../common/sshtools.py:113
#, python-format
msgid "Can't unmount sshfs %s"
msgstr "Не могу отмонтировать sshfs %s"

#: ../../common/sshtools.py:187
#, python-format
msgid ""
"Failed to unlock SSH private key:\n"
"Error: %s"
msgstr ""

#: ../../common/sshtools.py:194
msgid ""
"Could not unlock ssh private key. Wrong password or password not available "
"for cron."
msgstr ""

#: ../../common/sshtools.py:199
msgid "sshfs not found. Please install e.g. 'apt-get install sshfs'"
msgstr ""

#: ../../common/sshtools.py:217
#, python-format
msgid ""
"Password-less authentication for %(user)s@%(host)s failed. Look at 'man "
"backintime' for further instructions."
msgstr ""

#: ../../common/sshtools.py:229
#, python-format
msgid ""
"Cipher %(cipher)s failed for %(host)s:\n"
"%(err)s"
msgstr ""

#: ../../common/sshtools.py:254
#, python-format
msgid "%s not found in ssh_known_hosts."
msgstr ""

#: ../../common/sshtools.py:274
#, python-format
msgid ""
"Remote path exists but is not a directory:\n"
" %s"
msgstr ""

#: ../../common/sshtools.py:276
#, python-format
msgid ""
"Remote path is not writeable:\n"
" %s"
msgstr ""

#: ../../common/sshtools.py:278
#, python-format
msgid ""
"Remote path is not executable:\n"
" %s"
msgstr ""

#: ../../common/sshtools.py:280
#, python-format
msgid ""
"Couldn't create remote path:\n"
" %s"
msgstr ""

#: ../../common/sshtools.py:302
#, python-format
msgid "Ping %s failed. Host is down or wrong address."
msgstr ""

#: ../../common/sshtools.py:325 ../../common/sshtools.py:383
#, python-format
msgid ""
"Remote host %(host)s doesn't support '%(command)s':\n"
"%(err)s\n"
"Look at 'man backintime' for further instructions"
msgstr ""

#: ../../common/sshtools.py:387
#, python-format
msgid ""
"Check commands on host %(host)s returned unknown error:\n"
"%(err)s\n"
"Look at 'man backintime' for further instructions"
msgstr ""

#: ../../common/sshtools.py:400
#, python-format
msgid "Remote host %s doesn't support hardlinks"
msgstr ""

#: ../../gnome/app.py:185 ../../gnome/app.py:594 ../../gnome/app.py:608
#: ../../kde4/app.py:299 ../../kde4/app.py:570 ../../kde4/app.py:587
msgid "Done"
msgstr "Готово"

#: ../../gnome/app.py:189 ../../kde4/app.py:143 ../../kde4/app.py:159
#: ../../gnome/mainwindow.glade:225
msgid "Help"
msgstr "Справка"

#: ../../gnome/app.py:191 ../../kde4/app.py:146
#: ../../gnome/mainwindow.glade:223
msgid "Website"
msgstr "Сайт"

#: ../../gnome/app.py:192 ../../kde4/app.py:148
msgid "FAQ"
msgstr ""

#: ../../gnome/app.py:193 ../../kde4/app.py:150
msgid "Ask a question"
msgstr ""

#: ../../gnome/app.py:194 ../../kde4/app.py:152
msgid "Report a bug"
msgstr ""

#: ../../gnome/app.py:196 ../../kde4/app.py:155
msgid "About"
msgstr "О программе"

#: ../../gnome/app.py:202 ../../gnome/app.py:887
#: ../../gnome/restoredialog.py:84 ../../gnome/snapshotsdialog.py:83
#: ../../gnome/snapshotsdialog.py:244 ../../kde4/app.py:206
#: ../../kde4/app.py:216 ../../kde4/restoredialog.py:68
#: ../../kde4/snapshotsdialog.py:111 ../../kde4/snapshotsdialog.py:117
#: ../../gnome/mainwindow.glade:350 ../../gnome/mainwindow.glade:352
#: ../../gnome/snapshotsdialog.glade:169 ../../gnome/snapshotsdialog.glade:171
msgid "Restore"
msgstr "Восстановить"

#: ../../gnome/app.py:203 ../../gnome/app.py:888 ../../gnome/gnometools.py:67
#: ../../gnome/snapshotsdialog.py:84 ../../gnome/snapshotsdialog.py:245
#: ../../kde4/app.py:208 ../../kde4/restoredialog.py:42
#: ../../kde4/snapshotsdialog.py:113
msgid "Restore to ..."
msgstr ""

#: ../../gnome/app.py:205 ../../gnome/app.py:896 ../../gnome/app.py:1146
#: ../../kde4/app.py:1041
#, python-format
msgid "Restore '%s'"
msgstr ""

#: ../../gnome/app.py:207 ../../gnome/app.py:897 ../../gnome/app.py:1147
#: ../../kde4/app.py:1042
#, python-format
msgid "Restore '%s' to ..."
msgstr ""

#: ../../gnome/app.py:226 ../../kde4/app.py:235
msgid "Shortcuts"
msgstr "Ярлыки"

#: ../../gnome/app.py:249
msgid "Name"
msgstr "Название"

#: ../../gnome/app.py:260
msgid "Size"
msgstr "Размер"

#: ../../gnome/app.py:267
msgid "Date"
msgstr "Дата"

#: ../../gnome/app.py:286 ../../gnome/app.py:883
#: ../../gnome/snapshotsdialog.py:116 ../../kde4/app.py:171
#: ../../kde4/app.py:221 ../../kde4/snapshotsdialog.py:91
#: ../../gnome/mainwindow.glade:366 ../../gnome/mainwindow.glade:368
#: ../../gnome/snapshotsdialog.glade:8
msgid "Snapshots"
msgstr "Резервные копии"

#: ../../gnome/app.py:348 ../../kde4/app.py:362
msgid ""
"Can't find snapshots folder.\n"
"If it is on a removable drive please plug it and then press OK"
msgstr ""
"Невозможно найти каталог резервных копий!\n"
"Если он находится на съёмном диске, пожалуйста, подключите его и нажмите ОК"

#: ../../gnome/app.py:611 ../../kde4/app.py:590
msgid "Done, no backup needed"
msgstr "Всё сделано, сохранять ничего не надо"

#: ../../gnome/app.py:619 ../../kde4/app.py:598
msgid "Working:"
msgstr "Работаю:"

#: ../../gnome/app.py:623 ../../kde4/app.py:602
msgid "Error:"
msgstr "Ошибка:"

#: ../../gnome/app.py:636 ../../kde4/app.py:649
msgid "Global"
msgstr "Общее"

#: ../../gnome/app.py:637 ../../kde4/app.py:650
msgid "Root"
msgstr "Корневой каталог"

#: ../../gnome/app.py:638 ../../kde4/app.py:651
msgid "Home"
msgstr "Домашний каталог"

#: ../../gnome/app.py:662
msgid "Bookmarks"
msgstr "Закладки"

#: ../../gnome/app.py:675 ../../kde4/app.py:662
msgid "Backup folders"
msgstr "Резервные директории"

#: ../../gnome/app.py:701 ../../kde4/app.py:734
msgid "Today"
msgstr "Сегодня"

#: ../../gnome/app.py:705 ../../kde4/app.py:738
msgid "Yesterday"
msgstr "Вчера"

#: ../../gnome/app.py:709 ../../kde4/app.py:742
msgid "This week"
msgstr "Эта неделя"

#: ../../gnome/app.py:713 ../../kde4/app.py:746
msgid "Last week"
msgstr "На прошлой неделе"

#: ../../gnome/app.py:764 ../../kde4/app.py:398
msgid ""
"If you close this window Back In Time will not be able to shutdown your "
"system when the snapshot has finished.\n"
"Do you really want to close?"
msgstr ""

#: ../../gnome/app.py:881 ../../gnome/snapshotsdialog.py:238
msgid "Open"
msgstr "Открыть"

#: ../../gnome/app.py:1056 ../../kde4/app.py:90 ../../kde4/app.py:805
#: ../../gnome/mainwindow.glade:73 ../../gnome/mainwindow.glade:75
msgid "Snapshot Name"
msgstr "Название резервной копии"

#: ../../gnome/app.py:1075
#, python-format
msgid ""
"Are you sure you want to remove the snapshot:\n"
"<b>%s</b>"
msgstr ""
"Вы уверены в том, что хотите удалить резервную копию:\n"
"<b>%s</b>"

#: ../../gnome/app.py:1123 ../../kde4/app.py:1010
#, python-format
msgid "Snapshot: %s"
msgstr "Резервная копия: %s"

#: ../../gnome/app.py:1124 ../../kde4/app.py:1011
#, python-format
msgid "View the snapshot made at %s"
msgstr "Посмотреть резервную копию, которая сделана %s"

#: ../../gnome/logviewdialog.py:106 ../../gnome/settingsdialog.py:414
#: ../../kde4/logviewdialog.py:83 ../../kde4/settingsdialog.py:590
msgid "All"
msgstr "Все"

#: ../../gnome/logviewdialog.py:107 ../../gnome/settingsdialog.py:412
#: ../../kde4/logviewdialog.py:84 ../../kde4/settingsdialog.py:588
msgid "Errors"
msgstr "Ошибки"

#: ../../gnome/logviewdialog.py:112 ../../kde4/logviewdialog.py:89
msgid "Changes"
msgstr "Изменения"

#: ../../gnome/logviewdialog.py:115 ../../kde4/logviewdialog.py:92
msgid "Informations"
msgstr "Оповещения"

#: ../../gnome/settingsdialog.py:235 ../../kde4/settingsdialog.py:387
msgid "Include files and folders"
msgstr "Включить файлы и папки"

#: ../../gnome/settingsdialog.py:264
msgid "Exclude patterns, files or folders"
msgstr "Исключить шаблоны, файлы или папки"

#: ../../gnome/settingsdialog.py:411 ../../kde4/settingsdialog.py:587
msgid "None"
msgstr "Нет"

#: ../../gnome/settingsdialog.py:413 ../../kde4/settingsdialog.py:589
msgid "Changes & Errors"
msgstr "Изменения и ошибки"

#: ../../gnome/settingsdialog.py:458
msgid "Snapshots folder"
msgstr "Каталог резервных копий"

#: ../../gnome/settingsdialog.py:466 ../../kde4/settingsdialog.py:1314
msgid "Are you sure you want to change snapshots folder ?"
msgstr ""
"Вы уверены, что хотите изменить каталог для хранения резервных копий?"

#: ../../gnome/settingsdialog.py:901 ../../kde4/settingsdialog.py:914
msgid ""
"Custom Hours can only be a comma seperate list of hours (e.g. 8,12,18,23) or "
"*/3 for periodic backups every 3 hours"
msgstr ""

#: ../../gnome/settingsdialog.py:1141 ../../kde4/settingsdialog.py:653
msgid "New profile"
msgstr "Новый профиль"

#: ../../gnome/settingsdialog.py:1155 ../../kde4/settingsdialog.py:669
msgid "Rename profile"
msgstr "Переименовать профиль"

#: ../../gnome/settingsdialog.py:1167 ../../kde4/settingsdialog.py:683
#, python-format
msgid "Are you sure you want to delete the profile \"%s\" ?"
msgstr "Вы действительно хотите удалить профиль \"%s\"?"

#: ../../gnome/settingsdialog.py:1193 ../../kde4/settingsdialog.py:1278
msgid "Include file"
msgstr "Включить файл"

#: ../../gnome/settingsdialog.py:1235 ../../kde4/settingsdialog.py:1241
msgid "Exclude pattern"
msgstr "Исключать папку по шаблону"

#: ../../gnome/settingsdialog.py:1240 ../../kde4/settingsdialog.py:1251
msgid "Exclude patterns can't contain ':' char !"
msgstr "Исключение. Шаблоны не могут содержать символ ':' !"

#: ../../gnome/settingsdialog.py:1246 ../../kde4/settingsdialog.py:1257
msgid "Exclude file"
msgstr "Исключая файл"

#: ../../gnome/snapshotsdialog.py:240
msgid "Jump to"
msgstr ""

#: ../../gnome/snapshotsdialog.py:248 ../../kde4/snapshotsdialog.py:164
msgid "Diff"
msgstr "Различие"

#: ../../gnome/snapshotsdialog.py:277 ../../kde4/snapshotsdialog.py:339
msgid "You can't compare a snapshot to itself"
msgstr "Нельзя сравнивать резервную копию с саму с собой"

#: ../../gnome/snapshotsdialog.py:284 ../../kde4/snapshotsdialog.py:346
#, python-format
msgid "Command not found: %s"
msgstr "Команда не найдена: %s"

#: ../../gnome/snapshotsdialog.py:307 ../../kde4/snapshotsdialog.py:367
#, python-format
msgid ""
"Do you really want to delete \"%(file)s\" in snapshot \"%(snapshot_id)s?\n"
msgstr ""

#: ../../gnome/snapshotsdialog.py:310 ../../kde4/snapshotsdialog.py:370
#, python-format
msgid "Do you really want to delete \"%(file)s\" in %(count)d snapshots?\n"
msgstr ""

#: ../../gnome/snapshotsdialog.py:312 ../../kde4/snapshotsdialog.py:372
msgid "WARNING: This can not be revoked!"
msgstr ""

#: ../../gnome/snapshotsdialog.py:317 ../../kde4/snapshotsdialog.py:377
#, python-format
msgid "Exclude \"%s\" from future snapshots?"
msgstr ""

#: ../../kde4/app.py:86 ../../gnome/mainwindow.glade:58
#: ../../gnome/mainwindow.glade:60
msgid "Refresh snapshots list"
msgstr "Обновить список резервных копий"

#: ../../kde4/app.py:94 ../../gnome/mainwindow.glade:88
#: ../../gnome/mainwindow.glade:90
msgid "Remove Snapshot"
msgstr "Удалить резервную копию"

#: ../../kde4/app.py:98 ../../gnome/mainwindow.glade:105
msgid "View Snapshot Log"
msgstr "Просмотр журнала снимков состояния"

#: ../../kde4/app.py:102 ../../gnome/mainwindow.glade:121
msgid "View Last Log"
msgstr "Просмотреть последний журнал"

#: ../../kde4/app.py:108 ../../kde4/settingsdialog.py:69
#: ../../gnome/mainwindow.glade:144 ../../gnome/mainwindow.glade:146
#: ../../gnome/settingsdialog.glade:7
msgid "Settings"
msgstr "Настройки"

#: ../../kde4/app.py:124 ../../gnome/mainwindow.glade:169
msgid "Shutdown system after snapshot has finished."
msgstr ""

#: ../../kde4/app.py:131 ../../gnome/mainwindow.glade:184
#: ../../gnome/mainwindow.glade:186
msgid "Exit"
msgstr "Выйти"

#: ../../kde4/app.py:186 ../../gnome/mainwindow.glade:289
msgid "Up"
msgstr "Вверх"

#: ../../kde4/app.py:199 ../../gnome/mainwindow.glade:325
msgid "Show hidden files"
msgstr "Показать скрытые файлы"

#: ../../kde4/app.py:244
msgid ""
"This folder doesn't exist\n"
"in the current snapshot !"
msgstr ""
"Эта папка не существует\n"
"в данном снимке!"

#: ../../kde4/app.py:839
#, python-format
msgid ""
"Are you sure you want to remove the snapshot:\n"
"%s"
msgstr ""
"Вы уверены что вы хотите удалить резервную копию:\n"
"%s"

#: ../../kde4/app.py:1140 ../../kde4/kde4systrayicon.py:117
msgid "Working..."
msgstr "Выполняется..."

#: ../../kde4/logviewdialog.py:55
msgid "Error Log View"
msgstr "Просмотр журнала ошибок"

#: ../../kde4/logviewdialog.py:65 ../../kde4/settingsdialog.py:79
#: ../../kde4/settingsdialog.py:293 ../../gnome/logviewdialog.glade:73
#: ../../gnome/settingsdialog.glade:72
msgid "Profile:"
msgstr "Профиль:"

#: ../../kde4/logviewdialog.py:77 ../../gnome/logviewdialog.glade:110
msgid "Filter:"
msgstr "Фильтр:"

#: ../../kde4/logviewdialog.py:100 ../../gnome/logviewdialog.glade:169
msgid "[E] Error, [I] Information, [C] Change"
msgstr "[E] Ошибка, [I] Информация, [C] Изменение"

#: ../../kde4/logviewdialog.py:103 ../../gnome/logviewdialog.glade:195
msgid "decode paths"
msgstr ""

#: ../../kde4/settingsdialog.py:88
msgid "Edit"
msgstr "Изменить"

#: ../../kde4/settingsdialog.py:106 ../../gnome/settingsdialog.glade:1207
msgid "General"
msgstr "Общие"

#: ../../kde4/settingsdialog.py:114 ../../gnome/settingsdialog.glade:199
msgid "Mode:"
msgstr ""

#: ../../kde4/settingsdialog.py:127 ../../kde4/settingsdialog.py:1311
msgid "Where to save snapshots"
msgstr "Место для сохранения резервных копий"

#: ../../kde4/settingsdialog.py:146
msgid "SSH Settings"
msgstr ""

#: ../../kde4/settingsdialog.py:158 ../../kde4/settingsdialog.py:283
#: ../../gnome/settingsdialog.glade:323 ../../gnome/settingsdialog.glade:583
#: ../../gnome/settingsdialog.glade:874
msgid "Host:"
msgstr "Хост:"

#: ../../kde4/settingsdialog.py:163 ../../gnome/settingsdialog.glade:350
#: ../../gnome/settingsdialog.glade:610
msgid "Port:"
msgstr ""

#: ../../kde4/settingsdialog.py:168 ../../kde4/settingsdialog.py:288
#: ../../gnome/settingsdialog.glade:377 ../../gnome/settingsdialog.glade:637
#: ../../gnome/settingsdialog.glade:902
msgid "User:"
msgstr "Пользователь:"

#: ../../kde4/settingsdialog.py:173 ../../gnome/settingsdialog.glade:416
msgid "Path:"
msgstr ""

#: ../../kde4/settingsdialog.py:178 ../../gnome/settingsdialog.glade:459
msgid "Cipher:"
msgstr ""

#: ../../kde4/settingsdialog.py:184
msgid "Private Key:"
msgstr ""

#: ../../kde4/settingsdialog.py:228 ../../kde4/settingsdialog.py:237
#: ../../kde4/settingsdialog.py:243 ../../gnome/settingsdialog.glade:703
#: ../../gnome/settingsdialog.glade:743
msgid "Password"
msgstr ""

#: ../../kde4/settingsdialog.py:249 ../../gnome/settingsdialog.glade:775
msgid "Save Password to Keyring"
msgstr ""

#: ../../kde4/settingsdialog.py:252 ../../gnome/settingsdialog.glade:792
msgid "Cache Password for Cron (Security issue: root can read password)"
msgstr ""

#: ../../kde4/settingsdialog.py:264 ../../gnome/settingsdialog.glade:968
msgid "Advanced"
msgstr "Дополнительно"

#: ../../kde4/settingsdialog.py:276 ../../gnome/settingsdialog.glade:848
msgid "Auto Host / User / Profile Id"
msgstr "Автоматическая идентификация Компьютера / Пользователя / Профиля"

#: ../../kde4/settingsdialog.py:301
msgid "Schedule"
msgstr "Расписание"

#: ../../kde4/settingsdialog.py:311 ../../gnome/settingsdialog.glade:1092
msgid "Day:"
msgstr ""

#: ../../kde4/settingsdialog.py:322 ../../gnome/settingsdialog.glade:1079
msgid "Weekday:"
msgstr ""

#: ../../kde4/settingsdialog.py:333 ../../gnome/settingsdialog.glade:1066
msgid "Hour:"
msgstr "Часы:"

#: ../../kde4/settingsdialog.py:344 ../../gnome/settingsdialog.glade:1053
msgid "Hours:"
msgstr ""

#: ../../kde4/settingsdialog.py:353 ../../gnome/settingsdialog.glade:1014
msgid ""
"Run Back In Time periodically with anacron. This is useful if the computer "
"is not running regular."
msgstr ""

#: ../../kde4/settingsdialog.py:357 ../../gnome/settingsdialog.glade:1040
msgid "Frequency in days:"
msgstr ""

#: ../../kde4/settingsdialog.py:368 ../../gnome/settingsdialog.glade:1026
msgid ""
"Run Back In Time as soon as the drive is connected (only once every X "
"days).\n"
"You will be prompted for your sudo password."
msgstr ""

#: ../../kde4/settingsdialog.py:379 ../../gnome/settingsdialog.glade:1310
msgid "Include"
msgstr "Включить"

#: ../../kde4/settingsdialog.py:403 ../../kde4/settingsdialog.py:445
#: ../../gnome/settingsdialog.glade:1411
msgid "Add file"
msgstr "Добавить файл"

#: ../../kde4/settingsdialog.py:418 ../../gnome/settingsdialog.glade:1474
msgid "Exclude"
msgstr "Исключить"

#: ../../kde4/settingsdialog.py:421 ../../gnome/settingsdialog.glade:1328
msgid ""
"<b>Warning:</b> Wildcards ('foo*', '[fF]oo', 'fo?') will be ignored with "
"mode 'SSH encrypted'.\n"
"Only separate asterisk are allowed ('foo/*', 'foo/**/bar')"
msgstr ""

#: ../../kde4/settingsdialog.py:430
msgid "Highly recommended:"
msgstr "Очень рекомендуется:"

#: ../../kde4/settingsdialog.py:464 ../../gnome/settingsdialog.glade:1608
msgid "Older than:"
msgstr "Старее чем:"

#: ../../kde4/settingsdialog.py:478 ../../gnome/settingsdialog.glade:1624
msgid "If free space is less than:"
msgstr "Если свободного места меньше чем:"

#: ../../kde4/settingsdialog.py:490 ../../gnome/settingsdialog.glade:1642
msgid "If free inodes is less than:"
msgstr ""

#: ../../kde4/settingsdialog.py:514 ../../gnome/settingsdialog.glade:1697
msgid "Keep all snapshots for the last"
msgstr ""

#: ../../kde4/settingsdialog.py:517 ../../kde4/settingsdialog.py:522
#: ../../gnome/settingsdialog.glade:1705 ../../gnome/settingsdialog.glade:1729
msgid "day(s)"
msgstr ""

#: ../../kde4/settingsdialog.py:519 ../../gnome/settingsdialog.glade:1717
msgid "Keep one snapshot per day for the last"
msgstr ""

#: ../../kde4/settingsdialog.py:524 ../../gnome/settingsdialog.glade:1743
msgid "Keep one snapshot per week for the last"
msgstr ""

#: ../../kde4/settingsdialog.py:527
msgid "weeks(s)"
msgstr ""

#: ../../kde4/settingsdialog.py:529 ../../gnome/settingsdialog.glade:1769
msgid "Keep one snapshot per month for the last"
msgstr ""

#: ../../kde4/settingsdialog.py:532 ../../gnome/settingsdialog.glade:1781
msgid "month(s)"
msgstr ""

#: ../../kde4/settingsdialog.py:534 ../../gnome/settingsdialog.glade:1795
msgid "Keep one snapshot per year for all years"
msgstr ""

#: ../../kde4/settingsdialog.py:537 ../../gnome/settingsdialog.glade:1496
msgid "Don't remove named snapshots"
msgstr "Не удалять переименованные резервные копии"

#: ../../kde4/settingsdialog.py:546 ../../gnome/settingsdialog.glade:2104
msgid "Options"
msgstr "Опции"

#: ../../kde4/settingsdialog.py:549 ../../gnome/settingsdialog.glade:1920
msgid "Enable notifications"
msgstr "Включить уведомления"

#: ../../kde4/settingsdialog.py:552 ../../gnome/settingsdialog.glade:1937
msgid "Disable snapshots when on battery"
msgstr "Отключить создание резервных копий при работе от батареи"

#: ../../kde4/settingsdialog.py:555
msgid "Power status not available from system"
msgstr "Статус питания недоступен."

#: ../../kde4/settingsdialog.py:558 ../../gnome/settingsdialog.glade:1955
msgid "Backup files on restore"
msgstr "Сохранять файлы во время восстановления"

#: ../../kde4/settingsdialog.py:561 ../../gnome/settingsdialog.glade:1972
msgid "Continue on errors (keep incomplete snapshots)"
msgstr "Игнорировать ошибки (сохранять незавершенные копии)"

#: ../../kde4/settingsdialog.py:564 ../../gnome/settingsdialog.glade:1989
msgid "Use checksum to detect changes"
msgstr "Использовать контрольную сумму для обнаружения изменений"

#: ../../kde4/settingsdialog.py:567 ../../gnome/settingsdialog.glade:2006
msgid "Full rsync mode. May be faster but:"
msgstr ""

#: ../../kde4/settingsdialog.py:568 ../../gnome/settingsdialog.glade:2032
msgid ""
"- snapshots are no read-only\n"
"- destination file-system must support all linux attributes (dates, rights, "
"user, group ...)"
msgstr ""

#: ../../kde4/settingsdialog.py:575 ../../gnome/settingsdialog.glade:2045
msgid "Check for changes (don't take a new snapshot if nothing changed)"
msgstr ""

#: ../../kde4/settingsdialog.py:582
msgid "Log Level:"
msgstr "Уровень журналирования"

#: ../../kde4/settingsdialog.py:597 ../../gnome/settingsdialog.glade:2366
msgid "Expert Options"
msgstr "Расширенные настройки"

#: ../../kde4/settingsdialog.py:600
msgid "Change these options only if you really know what you are doing !"
msgstr "Изменяйте эти параметры, только если вы точно знаете, что делаете!"

#: ../../kde4/settingsdialog.py:608 ../../gnome/settingsdialog.glade:2133
msgid "Run 'nice' as cron job (default: enabled)"
msgstr "Запустить 'nice' как задание cron (по умолчанию: включено)"

#: ../../kde4/settingsdialog.py:611 ../../gnome/settingsdialog.glade:2150
msgid "Run 'ionice' as cron job (default: enabled)"
msgstr "Запустить  'ionice' как задачу в кроне (по умолчанию: отключено)"

#: ../../kde4/settingsdialog.py:614 ../../gnome/settingsdialog.glade:2167
msgid "Run 'ionice' when taking a manual snapshot (default: disabled)"
msgstr ""

#: ../../kde4/settingsdialog.py:617 ../../gnome/settingsdialog.glade:2184
msgid "Run 'nice' on remote host (default: disabled)"
msgstr ""

#: ../../kde4/settingsdialog.py:620 ../../gnome/settingsdialog.glade:2201
msgid "Run 'ionice' on remote host (default: disabled)"
msgstr ""

#: ../../kde4/settingsdialog.py:626 ../../gnome/settingsdialog.glade:2224
msgid "Limit rsync bandwidth usage: "
msgstr ""

#: ../../kde4/settingsdialog.py:629 ../../gnome/settingsdialog.glade:2259
msgid " KB/sec"
msgstr ""

#: ../../kde4/settingsdialog.py:634 ../../gnome/settingsdialog.glade:2280
msgid "Preserve ACL"
msgstr "Сохранять ACL"

#: ../../kde4/settingsdialog.py:637 ../../gnome/settingsdialog.glade:2297
msgid "Preserve extended attributes (xattr)"
msgstr "Сохранять дополнительные атрибуты (xattr)"

#: ../../kde4/settingsdialog.py:640 ../../gnome/settingsdialog.glade:2314
msgid "Copy unsafe links (works only with absolute links)"
msgstr ""
"Копировать небезопасные ссылки (работает только с абсолютными ссылками)"

#: ../../kde4/settingsdialog.py:643 ../../gnome/settingsdialog.glade:2331
msgid "Copy links (dereference symbolic links)"
msgstr "Копировать ссылки (разыменование символических ссылок)"

#: ../../kde4/snapshotsdialog.py:46 ../../kde4/snapshotsdialog.py:173
#: ../../gnome/snapshotsdialog.glade:385
msgid "Diff Options"
msgstr "Параметры сравнения"

#: ../../kde4/snapshotsdialog.py:56 ../../gnome/snapshotsdialog.glade:314
msgid "Command:"
msgstr "Команда:"

#: ../../kde4/snapshotsdialog.py:60 ../../gnome/snapshotsdialog.glade:322
msgid "Parameters:"
msgstr "Параметры:"

#: ../../kde4/snapshotsdialog.py:64 ../../gnome/snapshotsdialog.glade:334
msgid "Use %1 and %2 for path parameters"
msgstr "Использовать %1 и %2 для комманды"

#: ../../kde4/snapshotsdialog.py:123 ../../gnome/snapshotsdialog.glade:184
#: ../../gnome/snapshotsdialog.glade:186
msgid "Delete"
msgstr ""

#: ../../kde4/snapshotsdialog.py:128 ../../gnome/snapshotsdialog.glade:199
#: ../../gnome/snapshotsdialog.glade:201
msgid "Select All"
msgstr ""

#: ../../kde4/snapshotsdialog.py:132 ../../gnome/snapshotsdialog.glade:88
msgid "List only different snapshots"
msgstr "Показать только различающиеся снимки состояния"

#: ../../kde4/snapshotsdialog.py:139 ../../gnome/snapshotsdialog.glade:110
msgid "List only equal snapshots to: "
msgstr ""

#: ../../kde4/snapshotsdialog.py:149 ../../gnome/snapshotsdialog.glade:146
msgid "Deep check (more accurate, but slow)"
msgstr "Глубокая проверка (более точная, но медленная)"

#: ../../kde4/snapshotsdialog.py:174
msgid "Go To"
msgstr "Перейти к"

#: ../../gnome/logviewdialog.glade:9
msgid "Log View"
msgstr "Просмотр журнала"

#: ../../gnome/logviewdialog.glade:179
msgid "scroll down automatic"
msgstr ""

#: ../../gnome/mainwindow.glade:7
msgid "Back In Time"
msgstr "Back In Time"

#: ../../gnome/mainwindow.glade:45
msgid "Backup Now"
msgstr "Создать резервную копию сейчас"

#: ../../gnome/mainwindow.glade:171
msgid "Shutdown"
msgstr ""

#: ../../gnome/mainwindow.glade:287
msgid "Parent folder"
msgstr "Родительский каталог"

#: ../../gnome/mainwindow.glade:327
msgid "Hidden files"
msgstr "Скрытые файлы"

#: ../../gnome/mainwindow.glade:451
msgid ""
"<b>This folder doesn't exist\n"
"in the current snapshot !</b>"
msgstr ""
"Эта папка не существует\n"
"в данном снимке!</b>"

#: ../../gnome/settingsdialog.glade:287
msgid "<b>Where to save snapshots</b>"
msgstr "<b>Куда сохранить резервные копии</b>"

#: ../../gnome/settingsdialog.glade:483
msgid "Private key:"
msgstr ""

#: ../../gnome/settingsdialog.glade:534
msgid "Advanced SSH Settings"
msgstr ""

#: ../../gnome/settingsdialog.glade:552
msgid "<b>SSH Settings</b>"
msgstr ""

#: ../../gnome/settingsdialog.glade:815
msgid "<b>Password</b>"
msgstr ""

#: ../../gnome/settingsdialog.glade:930
msgid "Profile Id:"
msgstr "ID профиля:"

#: ../../gnome/settingsdialog.glade:1115
msgid ""
"hours seperate by comma ( e.g. 8,12,18,23) or */3 for periodic backups every "
"3 hours (cron style)"
msgstr ""

#: ../../gnome/settingsdialog.glade:1163
msgid "<b>Schedule</b>"
msgstr "<b>Расписание</b>"

#: ../../gnome/settingsdialog.glade:1363
msgid "<b>Highly recommended:</b>"
msgstr "<b>Очень рекомендуется:</b>"

#: ../../gnome/settingsdialog.glade:1377
msgid "label"
msgstr "метка"

#: ../../gnome/settingsdialog.glade:1755
msgid "week(s)"
msgstr ""

#: ../../gnome/settingsdialog.glade:2069
msgid "Log level:"
msgstr "Уровень журналирования:"

#: ../../gnome/settingsdialog.glade:2121
msgid ""
"<b>Change these options only if you really know what you are doing !</b>"
msgstr ""
"<b>Изменяйте эти параметры, только если вы точно знаете, что делаете!</b>"

#: ../../gnome/snapshotsdialog.glade:263
msgid "Diff with"
msgstr "Сравнивать с"

#: ../../gnome/textinputdialog.glade:7
msgid "Pattern"
msgstr "Шаблон"

#: ../../gnome/textinputdialog.glade:70
msgid "Text"
msgstr ""
